.github/workflows/test.yml

```diff
@@ -55,19 +115,36 @@ jobs:
             \"os\": [ \"ubuntu-latest\" ], \
             \"python-version\": [ \"2.7\", \"3.10\" ], \
             \"package_level\": [ \"minimum\", \"latest\" ], \
-            \"include\": [ \
+            \"exclude\": [ \
               { \
                 \"os\": \"ubuntu-latest\", \
-                \"python-version\": \"3.5\", \
+                \"python-version\": \"2.7\", \
                 \"package_level\": \"minimum\" \
               }, \
               { \
                 \"os\": \"ubuntu-latest\", \
+                \"python-version\": \"2,7\", \
+                \"package_level\": \"latest\" \
+              } \
```

Comment:
Instead of first specifying 2.7 on line 116 and then excluding all of its (two) matrix expansions again, 2.7 should simply not be specified on line 116.

Reply:
DONE